markdown/ranger/ranger-kerberos.html.md.erb

```diff
@@ -0,0 +1,209 @@
+- Noted the host name or IP address of your Ranger Administration host (\<ranger-admin-node\>) and HAWQ master (\<master\>) nodes.
+- Identified an existing Kerberos Key Distribution Center (KDC) or set up your KDC as described in [Install and Configure a Kerberos KDC Server](../clientaccess/kerberos.html#task_setup_kdc).
+    - Note the host name or IP address of your KDC (\<kdc-server\>).
+    - Note the name of the Kerberos \<realm\> in which your cluster resides.
+- Enabled Ranger Authorization for HAWQ. See [Configuring HAWQ to use Ranger Policy Management](ranger-integration-config.html).
+
+
+## Configure Ranger for Kerberized HAWQ<a id="ra2hawq_kerb_cfg"></a>
+
+When you define HAWQ Ranger authorization policies, the Ranger Administration Host uses JDBC to connect to HAWQ during policy definition to look up policy resource names. When Kerberos user authentication is enabled for HAWQ, you must configure this connection for Kerberos.
+
+To configure Ranger access to a HAWQ cluster enabled with Kerberos user authentication, you must:
+
+- Identify an existing HAWQ administrative role or create a new HAWQ administrative role for Ranger lookup of HAWQ resources
+- Create a Kerberos principal for the lookup role
+- Update the Ranger HAWQ service definition
+
+### Procedure <a id="kerb_ra2hawq_proc"></a>
+
+Perform the following procedure to enable the Ranger Administration Host to look up resources in your kerberized HAWQ cluster. You will perform operations on the HAWQ \<master\>, \<ranger-admin-node\>, and \<kdc-server\> nodes.
+
+1. Log in to the HAWQ master node and set up your environment:
+
+    ``` shell
+    $ ssh gpadmin@<master>
+    gpadmin@master$ . /usr/local/hawq/greenplum_path.sh
+    ```
+
+2. Identify an existing or create a new HAWQ administrative role for Ranger resource lookup. For example, to create a new administrative role:
+
+    ``` shell
+    gpadmin@master$ psql -c 'CREATE ROLE "rangerlookup_hawq" with LOGIN SUPERUSER;' 
+    ```
+   
+    You may choose a different name for the Ranger lookup role.
+
+3. Log in to the KDC server system and generate a principal for the HAWQ `rangerlookup_hawq` role. Substitute your Kerberos \<realm\>. For example:
+
+    ``` shell
+    $ ssh root@<kdc-server>
+    root@kdc-server$ kadmin.local -q "addprinc -pw changeme rangerlookup_hawq@REALM.DOMAIN"
+    ```
+    
+    You do not need to generate a keytab file for the `rangerlookup_hawq` principal because you will provide the password in the HAWQ service definition of the Ranger Admin UI.
+
+4. Start the Ranger Admin UI in a supported web browser. The default URL is \<ranger-admin-node\>:6080. 
+
+5. Locate the HAWQ service definition and press the **Edit** button. 
+
+6. Update the applicable **Config Properties** fields:
+
+    **HAWQ User Name*** - The HAWQ Ranger lookup role you identified or created in Step 2 above.  
```

Comment:
I think for consistency we should change the hyphens to colons here.  Also use sentences here:  "Enter the HAWQ Ranger lookup role ..."